redis/src/cluster_async/mod.rs

```diff
@@ -794,7 +794,12 @@ impl<C> Future for Request<C> {
                         .into();
                     }
                 };
-                trace!("Request error `{}` on node `{:?}", err, address);
+
+                warn!(
```

Comment:
Does the user have a way to know how it will be handle?
Otherwise it can be a bit confusing.
Can we ad somehow some info and how it will be handle? or doc somewhere else the error handling and refer to it?